src/main/java/seedu/address/model/profile/course/module/Module.java

```diff
@@ -1,5 +1,7 @@
 package seedu.address.model.profile.course.module;
 
+import seedu.address.TaskList;
```

Comment:
Shift TaskList to same package as Module, then this import can be removed!

Reply:
Got it, thanks guangyan!